modules/fedimint-wallet-common/src/config.rs

```diff
@@ -151,7 +155,10 @@ impl WalletConfig {
         );
 
         Self {
-            local: WalletConfigLocal { bitcoin_rpc },
+            local: WalletConfigLocal {
+                bitcoin_rpc,
```

Comment:
**question (blocking):** @elsirion it's unclear to me if this is OK. Are we sure we are not storing `WalletConfigLocal` anywhere?

Maybe `out_peer_id` should be stored in a separate field so we avoid storing it twice (and breaking compatibility)? If so, maybe we should do it on `master` too?

Reply:
See  #3367